lib/mvi/service.rb

```diff
@@ -78,6 +78,18 @@ def self.formatted_response(body)
         )
       }
     end
+
+    # MVI correlation id source id relationships:
+    # {source id}^{id type}^{assigning authority}^{assigning facility}^{id status}
+    # TODO(AJD): MVI team will be sending the mapping of system identifiers to
+    # va systems (e.g. 200VETS = vets.gov, 516 = ?) when we have that we can symbolize the keys
+    #
+    def self.map_correlation_ids(ids)
```

Comment:
this is begging to be a private method, but since it is a class method you'd have to manually make it a private method. I'm wondering if maybe this whole class could be refactored to be an instance of a singleton class and then would not need all the class methods?